app/tray_icon.js

```diff
@@ -11,8 +11,19 @@ let trayContextMenu = null;
 let tray = null;
 
 function createTrayIcon(getMainWindow, messages) {
-  // A smaller icon is needed on macOS
-  const iconSize = process.platform === 'darwin' ? '16' : '256';
+  let iconSize;
+  switch (process.platform) {
+    case 'darwin':
+      iconSize = '16';
+      break;
+    case 'win32':
+      iconSize = '32';
```

Comment:
I just realized there's a problem with this: `iconSize` is used further down:

https://github.com/signalapp/Signal-Desktop/blob/53f757dac80ca5d4027494dd561ce79c5bb04a88/app/tray_icon.js#L105

This corresponds to the [`images/alert` directory](https://github.com/signalapp/Signal-Desktop/tree/53f757dac80ca5d4027494dd561ce79c5bb04a88/images/alert), which only has folders for 16x16 and 256x256. I think we'll need to creates (and populate) `images/alert/32` or find an alternative. Maybe Electron has a way of doing this natively?